SU2_CFD/include/solvers/CScalarSolver.hpp

```diff
@@ -48,11 +48,13 @@ class CScalarSolver : public CSolver {
 
   unsigned long omp_chunk_size; /*!< \brief Chunk size used in light point loops. */
 
-  su2double lowerlimit[MAXNVAR] = {0.0}, /*!< \brief contains lower limits for turbulence variables. */
-      upperlimit[MAXNVAR] = {0.0};       /*!< \brief contains upper limits for turbulence variables. */
+  su2double lowerlimit[MAXNVAR] = {std::numeric_limits<su2double>::lowest()}; /*!< \brief contains lower limits for turbulence variables. Note that ::min() returns the smallest positive value for floats. */
+  su2double upperlimit[MAXNVAR] = {std::numeric_limits<su2double>::max()}; /*!< \brief contains upper limits for turbulence variables. */
```

Comment:
This does not do what you think it does :) after the first you have zeros.

Reply:
hm, yes. it didn't break on me because SA only has 1 var (but some solver in the future might have more and does not explicitly set those). I'll fix it

Reply:
Now it is done via a loop in the CScalarVar ctor, so each derived solver can overwrite that